content/src/repository/extensions/DeploymentPointerChangesRepository.ts

```diff
@@ -21,26 +23,34 @@ export class DeploymentPointerChangesRepository {
   }
 
   async getPointerChangesForDeployments(
-    deploymentIds: DeploymentId[]
-  ): Promise<Map<DeploymentId, Map<Pointer, { before: EntityId | undefined; after: DELTA_POINTER_RESULT }>>> {
+    deploymentIds: DeploymentId[],
+    fields: PointerChangesField[] = []
+  ): Promise<
+    Map<DeploymentId, Map<Pointer, { before?: EntityId; after: DELTA_POINTER_RESULT; authChain?: AuthChain }>>
+  > {
     const result: Map<
       DeploymentId,
-      Map<Pointer, { before: EntityId | undefined; after: DELTA_POINTER_RESULT }>
+      Map<Pointer, { before: EntityId | undefined; after: DELTA_POINTER_RESULT; authChain?: AuthChain }>
     > = new Map()
     if (deploymentIds.length > 0) {
       const deltas = await this.db.any(
         `
-              SELECT deployment, pointer, after, deployments.entity_id AS before
+              SELECT deployment, pointer, after, deployments.entity_id AS before, deployments.auth_chain AS "authChain"
```

Comment:
I had several different problems with case sensitive columns in postgres. I want to make sure this was double-checked or tested with integration tests. Otherwise I suggest to use `auth_chain` and change the casing by code in JS